version/version.go

```diff
@@ -0,0 +1,45 @@
+package version
+
+import (
+	"github.com/arduino/arduino-cli/i18n"
+)
+
+var (
+	defaultVersionString = "0.0.0-git"
+	versionString        = ""
+	commit               = ""
+	status               = ""
+	date                 = ""
+	tr                   = i18n.Tr
+)
+
+// Info FIXMEDOC
```

Comment:
I would implement this comment instead of leaving it blank

Reply:
What do you mean?

Reply:
This comment is used to avoid errors by check-go workflow. Instead of putting a comment to only avoid this, i would explain in a comment what's going on